Translation corrections (#406)
07th-mod/higurashi-console-arcs: Update/omot_010_3.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100836", 256, TRUE);
	OutputLine(NULL, "「ああ、それは悪いことじゃない。",

> 			NULL, "\"Right, there's nothing wrong about that. ", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Right, there's nothing wrong with that. ", Line_WaitForInput);
```

			NULL, "\"Why would I lie about that?", Line_WaitForInput);
	ModPlayVoiceLS(4, 2, "ps3/s14/02/210200223", 256, TRUE);
	OutputLine(NULL, "　リナさんも、お父さんも、レナのために色々と無理して、気を遣ってくれている。",
			NULL, " Rina-san and Dad are already pushing themselves to be considerate of me.", Line_WaitForInput);
	ModPlayVoiceLS(4, 2, "ps3/s14/02/210200224", 256, TRUE);
	OutputLine(NULL, "だからレナも、少しくらいは我慢しなくちゃ、って思ってる」",

> 			NULL, " So, Rena figures she put up with it a little for their sake.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " So, Rena figures she should put up with it a little for their sake.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 2, "ps3/s14/02/210200229", 256, TRUE);
	OutputLine(NULL, "「みんな我慢してるのに、それをぶち壊しにきたの？",

> 			NULL, "\"Even though everyone is doing their best to show restraint, you came to tear that apart?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Even though everyone is doing their best to be patient, you came to tear that apart?", Line_WaitForInput);
```
